catalog/EDF/projects_EDF_MethaneSAT_public-preview_L3concentration.jsonnet

```diff
@@ -0,0 +1,297 @@
+// This is a generic template for an Earth Engine ee.ImageCollection dataset
+// STAC Collection entry. After replacing all the values, remove the
+// explanatory comments, but keep the "TODO" comment.
+//
+// This template covers the basic case with just one Jsonnet file defining
+// the dataset. For more complex examples with templating, look for directories
+// with `.libsonnet` files.
+//
+// See https://jsonnet.org/ for more documentation on the Jsonnet language used
+// here to create STAC JSON.
+//
+// Coding style:
+// - Indenting should be two spaces.
+// - Do not use tabs.
+// - Limit line length to around 80 characters unless it breaks URL strings.
+// - Use only ASCII characters. \n and 32..176 decimal range.
+// - Use American English.
+//
+// STAC Specifications:
+//   - Overall: https://github.com/radiantearth/stac-spec
+//   - Extensions: https://github.com/stac-extensions/stac-extensions.github.io
+//     - Electro-Optical: https://github.com/stac-extensions/eo
+//     - Synthetic Aperture Radar: https://github.com/stac-extensions/sar
+//     - Scientific: https://github.com/stac-extensions/scientific
+//     - Version: https://github.com/stac-extensions/version
+
+// This jsonnet file has to be named using the following convention:
+//   <subdir>/<id_with_slashes_replaced_with_underbars>.jsonnet.
+
+// The asset id as used in Earth Engine:
+//   ee.ImageCollection('TEMPLATE/IMAGE_COLLECTION_V2_3');
+local id = 'projects/EDF/MethaneSAT/public-preview/L3concentration';
+
+// The directory under 'catalog' that contains the dataset.
+// For datasets under 'projects', leave off the 'projects' component.
+//   E.g., the 'projects/planet-nicfi/assets/basemaps/africa' asset is in the
+//   'planet-nicfi' subdirectory.
+local subdir = 'EDF';
```

Comment:
This should be your project name (edf-methanesat?)

Reply:
done.  moved files to new location, will resolve this and we can continue over there.